components/expenses/PayoutBankInformationForm.js

```diff
@@ -362,13 +362,10 @@ const PayoutBankInformationForm = ({ isNew, getFieldName, host, fixedCurrency, i
   // Display spinner if loading
   if (loading) {
     return <StyledSpinner />;
-  } else if (host.slug === TW_API_COLLECTIVE_SLUG && !data?.host?.transferwise) {
+  } else if (!host?.transferwise) {
```

Comment:
What's the rationale removing the previous restriction to `host.slug === TW_API_COLLECTIVE_SLUG` ?

Reply:
Ops sorry, this is me being a bit confused. We need both of these. I've added it back. Should have been more careful. 🤯